### ISSUE TITLE
I think the 'Notes' should include 'reverse' or 'reverse by responder'

### ISSUE THREAD

**Issue by ADavidBailey** [EdwardPiwowar/BBA#848]
0980F5818A2D52DB46FD6D6F3072
![Screenshot 2024-09-13 at 4 04 19 PM](https://github.com/user-attachments/assets/17ae703a-a900-48a6-93c7-48751e606d04)

**Comment by ThorvaldAagaard** [comment]
I am not sure as reverse is not a convention, but what is important and not in the description of the bid is that the first bid suit always is longer than the second bid suit, then I can live with a note of biddable suit.

This is how GIB defines it over 1N, but plauing NMF it is different
![image](https://github.com/user-attachments/assets/bf8d8557-37c1-468a-a84c-4a9dea4d33f6)

So better to look at this
![image](https://github.com/user-attachments/assets/477e9903-deba-4415-a6ee-022a88e70041)

**Comment by ADavidBailey** [comment]
It's not a big deal; but, I would like to have a lot more 'notes'.  I would like to know Jacoby Transfer, Texas Transfer, Super-Accept, Reverse, 1NT, etc.  This would make the summary of notes more useful to me.

If I have a scenario that I expect to open 1N, I would like to know the % of 1N opener's I'm getting.  For example, I would like to verify that my Reverse_By_Respoder is getting a high percentage of reverses by responder.  Here's my summary:

   15  Blackwood 0314, for !c
    2  Blackwood 0314, for !d
    4  Cue bid, a !h stopper
    3  Cue bid, a !s stopper
   12  Gerber, for n
    9  King ask
    3  King ask by 5nt
    1  Quantitative 4nt
    5  Splinter
    3  Unusual 1nt

**Comment by ThorvaldAagaard** [comment]
Yes, you are right, might also be a help for Edward when trying to fix some bidding errors.
